### ISSUE TITLE
Building from pypi fails due to missing subdirectory

### ISSUE THREAD

**Issue by benjamb** [lief-project/LIEF#438]
**Describe the bug**

As mentioned in #409, building from pypi sources fails. This is due to the manifest not including the `package/` subdirectory. This was added in a commit after the 0.10.1 tag: 0b205665c3d86fb1a01f149afcdfe8ec8e5a0724.

**To Reproduce**

```
wget https://pypi.io/packages/source/l/lief/lief-0.10.1.tar.gz
mkdir -p /tmp/
tar -C /tmp -xvf lief-0.10.1.tar.gz
cd /tmp/lief-0.10.1
python -m pip install . -vv
```

Build will error with:
```
  CMake Error at CMakeLists.txt:541 (add_subdirectory):
    add_subdirectory given source "/tmp/pip-req-build-n56pv69j/package" which
    is not an existing directory.
```

**Expected behavior**

Build should succeed.

**Environment (please complete the following information):**

 - System and Version: Debian Buster
 - Target format: N/A
 - LIEF commit version: 0.10.1

**Additional context**

Building from pypi is the recommended method for integration into conda-forge, though it currently builds using the source archive from github.

**Comment by toothbrush7777777** [comment]
I have the exact same issue, except installing with `pip`.

**Comment by romainthomas** [comment]
@benjamb it should be fixed in the next Pypi release thanks to your PR

**Comment by swilcox3** [comment]
I'm having the same issue, so I followed the advice to install the nightly release.  pip succeeds, but then I can't import the package.  
```
core@WINDOWSCI-01 C:\Users\core\scoop\apps\python\3.8.2\Lib\site-packages>pip3 --no-cache-dir install --index-url https://lief-project.github.io/packages lief
Looking in indexes: https://lief-project.github.io/packages
Collecting lief
  Downloading https://lief-project.github.io/packages/lief/lief-0.11.0.dev0-cp38-cp38-win32.whl (2.1 MB)
     |████████████████████████████████| 2.1 MB 6.4 MB/s
Installing collected packages: lief
Successfully installed lief-0.11.0.dev0

core@WINDOWSCI-01 C:\Users\core\scoop\apps\python\3.8.2\Lib\site-packages>python3
Python 3.8.2 (tags/v3.8.2:7b3ab59, Feb 25 2020, 23:03:10) [MSC v.1916 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import lief
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'lief'
>>>
```

I'm on Windows 10.

**Comment by romainthomas** [comment]
@swilcox3 Your installation is wrong. ``pip3`` and ``python3`` don't point to the same version of Python. Check with ``python3 -m pip install``

**Comment by swilcox3** [comment]
I've only got one version of Python 3 on this machine.
```
core@WINDOWSCI-01 C:\Users\core\scoop\apps\python\3.8.2\Lib\site-packages>python3 -m pip install lief
Requirement already satisfied: lief in c:\users\core\scoop\apps\python\3.8.2\lib\site-packages (0.11.0.dev0)
WARNING: You are using pip version 20.2.4; however, version 20.3 is available.
You should consider upgrading via the 'C:\Users\core\scoop\apps\python\current\python.exe -m pip install --upgrade pip' command.

core@WINDOWSCI-01 C:\Users\core\scoop\apps\python\3.8.2\Lib\site-packages>python3
Python 3.8.2 (tags/v3.8.2:7b3ab59, Feb 25 2020, 23:03:10) [MSC v.1916 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import lief
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'lief'
>>>
```

**Comment by romainthomas** [comment]
I just tested on Windows 10 and it works. I think that your problem is related to your environment, not this issue

**Comment by romainthomas** [comment]
Fixed with https://github.com/lief-project/LIEF/commit/0b205665c3d86fb1a01f149afcdfe8ec8e5a0724 (Will be available in the ``0.11.0``)
